vividus-plugin-excel/src/main/java/org/vividus/steps/ExcelResponseValidationSteps.java

```diff
@@ -136,11 +140,11 @@ private void reportResults(Stream<String> ranges, List<Entry<CellValue, Optional
         });
     }
 
-    private IExcelSheetsExtractor getExtractor()
+    private IExcelSheetsExtractor getExtractor(DataWrapper excelDoc)
     {
         try
         {
-            return new ExcelSheetsExtractor(httpTestContext.getResponse().getResponseBody());
```

Comment:
- `httpTestContext` is not used anymore ->
- `ExcelResponseValidationSteps` -> `ExcelDocumentValidationSteps` ->
- `vividus-plugin-rest-api` dependency can be dropped, can't it?